shellhub/__init__.py

```diff
@@ -1,5 +1,5 @@
 # Increment versions here according to SemVer
-__version__ = "0.0.1"
+__version__ = "0.0.4"
```

Comment:
Since we currently are at version `0.0.1`, `0.0.2` would be fine :)

Reply:
Ok

Reply:
Can you make it 0.0.2 ?

Reply:
You can edit your commit to change it again to `0.0.2` :)

Otherwise, use the suggestion
```suggestion
__version__ = "0.0.2"
```

If you want more info on versioning: semver.org